components/Cards/examples/supplemental/CardEditReorderCollectionCell.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  CardEditReorderCollectionCell.swift
+//  CatalogByConvention
+//
+//  Created by Galia Kaufman on 7/6/18.
+//
+//  Card images source:
+//  Material Design Guidelines (https://material.io/design/components/cards.html#card-collections)
+//
+
+import UIKit
+import MaterialComponents.MaterialCards
+
+class CardEditReorderCollectionCell: MDCCardCollectionCell {
+
+  var colorScheme = MDCSemanticColorScheme()
+  var typographyScheme = MDCTypographyScheme()
+
+  private lazy var imageView: UIImageView = {
+    let imageView = UIImageView(frame: CGRect.zero)
+    imageView.contentMode = .scaleAspectFit
+    imageView.layer.cornerRadius = 6.0
+    imageView.translatesAutoresizingMaskIntoConstraints = false
+    return imageView
+  }()
+
+  private lazy var titleLabel: UILabel = {
+    let label = UILabel(frame: CGRect.zero)
+    label.numberOfLines = 1
+    label.textAlignment = .center
+    label.lineBreakMode = .byTruncatingTail
+    label.font = self.typographyScheme.caption
+    label.translatesAutoresizingMaskIntoConstraints = false
+    return label
+  }()
+
+  func configure(title: String, number: Int) {
```

Comment:
I think it would be better for the CollectionViewController to set the image property instead. It seems a little strange for the view to know how to select its image.

Reply:
Makes sense.